Backend/Tools/rssFeedXMLParser.py

```diff
@@ -0,0 +1,47 @@
+from xml.etree.ElementTree import Element
```

Comment:
Why is this file called rssFeedXMLParser if it doesn't use rssFeed data?

Reply:
this function will be moved to the main XMLParser, where rssFeed Data is given as requestData through function reformatData

Reply:
after I move this, I'll add all needed imports